swarm/src/connection.rs

```diff
@@ -153,8 +155,9 @@ where
     max_negotiating_inbound_streams: usize,
     /// Contains all upgrades that are waiting for a new outbound substream.
     ///
-    /// The upgrade timeout is already ticking here so this may fail in case the remote is not
+    /// The upgrade timeout is already ticking here so this may fail in case the remote is not quick
     /// quick enough in providing us with a new stream.
```

Comment:
```suggestion
    /// The upgrade timeout is already ticking here so this may fail in case the remote is not
    /// quick enough in providing us with a new stream.
```